packages/vsx-registry/src/browser/vsx-extensions-contribution.ts

```diff
@@ -95,6 +99,12 @@ export class VSXExtensionsContribution extends AbstractViewContribution<VSXExten
             execute: () => this.installFromVSIX()
         });
 
+        commands.registerCommand(VSXExtensionsCommands.INSTALL_VSIX,
+            UriAwareCommandHandler.MonoSelect(this.selectionService, {
+                execute: fileURI => this.installVSIX(fileURI)
```

Comment:
Since we already know this will only work on *.vsix files, we should add the `isEnabled` (and maybe `isVisible`) condition here or was there a particular reason you only restricted that on the menu action?

Reply:
Or you mean someone can execute it from the command palette? This is a good point, I need to check it.

Reply:
Yes. If this is only supposed to be called programmatically, I think you can hide the command by not providing a label and only provide the label in the menu action. Again, if that is the purpose please add a short comment because it is definitely non-obvious ;-)

Reply:
I was thinking about using the command from the command palette and I like the idea.
I've added `isEnabled`  to the command registration, so now one can install vsix files from the explorer view without using a mouse. I know there are "keyboard first" fans out there and they will like it, I hope.